src/Listener/SaveSuspensionToDatabase.php

```diff
@@ -53,7 +50,7 @@ public function handle(Saving $event)
             $user = $event->user;
             $actor = $event->actor;
 
-            $this->assertCan($actor, 'suspend', $user);
+            $actor->can('suspend', $user);
```

Comment:
This should be $actor->assertCan

Reply:
Ah crap, my IDE is behind and doesn't reconize assertCan because my core version is several PRs behind. I have to get that resolved. A bunch of other PRs I just did are likely affected by the same issue.